lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -416,68 +428,108 @@ spec = do
+                eventually "pool worker finds the certificate" $ do
+                    r <- listPools ctx
+                    expectResponseCode HTTP.status200 r
+                    verify r
+                        [ expectListField 0
+                            (#cost) (`shouldBe` Just (Quantity 0))
+                        , expectListField 1
+                            (#cost) (`shouldBe` Just (Quantity 0))
+                        , expectListField 2
+                            (#cost) (`shouldBe` Just (Quantity 0))
+
+                        , expectListField 0
+                            #margin (`shouldBe` Just
+                                (Quantity $ unsafeMkPercentage 0.1 ))
+                        , expectListField 1
+                            #margin (`shouldBe` Just
+                                (Quantity $ unsafeMkPercentage 0.1 ))
+                        , expectListField 2
+                            #margin (`shouldBe` Just
+                                (Quantity $ unsafeMkPercentage 0.1 ))
+
+                        -- TODO: Switch from (.>=) to (.>) (and perhaps an eventually) when
+                        -- our stake pool cluster are producing blocks.
+                        , expectListField 0
+                            (#metrics . #producedBlocks) (.>= Quantity 0)
+                        , expectListField 1
+                            (#metrics . #producedBlocks) (.>= Quantity 0)
+                        , expectListField 2
+                            (#metrics . #producedBlocks) (.>= Quantity 0)
+                        -- TODO: Test that we have non-zero non-myopic member
+                        -- rewards, and sort by it.
+                        ]
+
+            it "contains pool metadata" $ \ctx -> do
+                eventually "metadata is fetched" $ do
+                    r <- listPools ctx
+                    let metadataList =
+                            [ StakePoolMetadata
+                                { ticker = (StakePoolTicker "GPA")
+                                , name = "Genesis Pool A"
+                                , description = Nothing
+                                , homepage = "https://iohk.io"
+                                }
+                            , StakePoolMetadata
+                                { ticker = (StakePoolTicker "GPB")
+                                , name = "Genesis Pool B"
+                                , description = Nothing
+                                , homepage = "https://iohk.io"
+                                }
+                            , StakePoolMetadata
+                                { ticker = (StakePoolTicker "GPC")
+                                , name = "Genesis Pool C"
+                                , description = Just "Lorem Ipsum Dolor Sit Amet."
+                                , homepage = "https://iohk.io"
+                                }
+                            ]
```

Comment:
Maybe we could avoid hard-coding the list here since it's available as `operators` in `Cardano.Wallet.Shelley.Launch`.

Reply:
Don't think we can (or want) `cardano-wallet-core-integration` depend on `cardano-wallet-shelley`… :S  We _could_ move `API.Shelley.StakePools` to `cardano-wallet-shelley:integration` though?

Reply:
> cardano-wallet-core-integration depend on cardano-wallet-shelley

Arf. I didn't realize that. That's quite lame indeed.